sonar-batch/src/main/java/org/sonar/batch/phases/DatabaseLessPhaseExecutor.java

```diff
@@ -49,12 +50,13 @@
   private final LocalIssueTracking localIssueTracking;
   private final ReportPublisher reportPublisher;
 
-  public DatabaseLessPhaseExecutor(Phases phases, InitializersExecutor initializersExecutor, SensorsExecutor sensorsExecutor,
+  public DatabaseLessPhaseExecutor(Phases phases, InitializersExecutor initializersExecutor, PostJobsExecutor postJobsExecutor, SensorsExecutor sensorsExecutor,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Method has 15 parameters, which is greater than 7 authorized.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)